assets/js/blocks/active-filters/block.js

```diff
@@ -0,0 +1,130 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { useQueryStateByKey } from '@woocommerce/base-hooks';
+import { useMemo, Fragment } from '@wordpress/element';
+import classnames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+import { getAttributeFromTaxonomy } from '../../utils/attributes';
+import { formatPriceRange, renderItem } from './utils';
+import ActiveAttributeFilters from './active-attribute-filters';
+
+/**
+ * Component displaying active filters.
+ */
+const ActiveFiltersBlock = ( {
+	attributes: blockAttributes,
+	isPreview = false,
+} ) => {
+	const [ productAttributes, setProductAttributes ] = useQueryStateByKey(
+		'product-grid',
+		'attributes',
+		[]
+	);
+	const [ minPrice, setMinPrice ] = useQueryStateByKey(
+		'product-grid',
+		'min_price'
+	);
+	const [ maxPrice, setMaxPrice ] = useQueryStateByKey(
+		'product-grid',
+		'max_price'
+	);
+
+	const activePriceFilters = useMemo( () => {
+		if ( ! Number.isFinite( minPrice ) && ! Number.isFinite( maxPrice ) ) {
+			return;
+		}
+		return renderItem(
+			__( 'Price:', 'woo-gutenberg-products-block' ),
+			formatPriceRange( minPrice, maxPrice ),
+			() => {
+				setMinPrice( null );
+				setMaxPrice( null );
+			}
+		);
+	}, [ minPrice, maxPrice, formatPriceRange ] );
+
+	const activeAttributeFilters = useMemo( () => {
+		return (
+			<Fragment>
+				{ productAttributes.map( ( attribute ) => {
+					const attributeObject = getAttributeFromTaxonomy(
+						attribute.attribute
+					);
+					return (
+						<ActiveAttributeFilters
+							attributeObject={ attributeObject }
+							slugs={ attribute.slug }
+							key={ attribute.attribute }
+						/>
+					);
+				} ) }
+			</Fragment>
+		);
+	}, [ productAttributes ] );
+
+	const hasFilters = useMemo( () => {
+		return (
+			productAttributes.length > 0 ||
+			Number.isFinite( minPrice ) ||
+			Number.isFinite( maxPrice )
+		);
+	}, [ productAttributes, minPrice, maxPrice ] );
+
+	if ( ! hasFilters && ! isPreview ) {
+		return null;
+	}
+
+	const TagName = `h${ blockAttributes.headingLevel }`;
+	const listClasses = classnames(
+		'wc-block-active-filters-list',
+		blockAttributes.displayStyle === 'chips' &&
+			'wc-block-active-filters-list--chips'
+	);
```

Comment:
Just noting that one neat syntax of `classnames` is using object notation.  You could do the same thing as above via:

```js
const listClasses = classnames( 
    'wc-block-active-filters-list',
    { 'wc-block-active-filters-list--chips': blockAttributes.displayStyle === 'chips' }
);
```
Not really much of a difference in this particular class, but useful if there's a lot of conditions for the generated classes.  Just sharing in case you weren't aware :)

Reply:
I was aware, I prefer the && notation but Ill swap it.